crates/biome_service/src/file_handlers/mod.rs

```diff
@@ -558,6 +559,44 @@ pub(crate) fn is_diagnostic_error(
     severity >= Severity::Error
 }
 
+/// Parse the "lang" attribute from the opening tag of the "\<script\>" block in Svelte or Vue files.
+/// This function will return the language based on the existence or the value of the "lang" attribute.
+/// We use the JSX parser at the moment to parse the opening tag. So the opening tag should be first
+/// matched by regular expressions.
+///
+/// TODO: We should change the parser when HTMLish languages are supported.
```

Comment:
```suggestion
// TODO: We should change the parser when HTMLish languages are supported.
```

The `TODO` shouldn't be a doc comment